### PR TITLE
adding save_animation example

### PR DESCRIPTION
**Pull request by braingram**
Adding a save_animation example in examples/basics/scene/. Uses imageio to save a few rendered frames from the sphere example in a gif

### CODE REVIEW

File: examples/basics/scene/save_animation.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                               method='cube', parent=view.scene,
+                               edge_color='black')
+
+sphere1.transform = STTransform(translate=[-2.5, 0, 0])
+sphere3.transform = STTransform(translate=[2.5, 0, 0])
+
+view.camera.set_range(x=[-3, 3])
+
+writer = imageio.get_writer('animation.gif')
+for i in xrange(n_steps * 2):
```
Comment (almarklein) [inline examples/basics/scene/save_animation.py]: You have a typo here, this must be `range` ;)

**Comment by almarklein** [comment]
Nice! It works fine here (Win10). Other than my small comment to make it work on Py3k, this LGTM.

**Comment by braingram** [comment]
Oops. Yeah, this was only tested on python 3. Let me know if there's anything else.

**Comment by Eric89GXL** [comment]
Thanks @braingram

**Pull request merged**